openshift-container-platform-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -89,6 +89,25 @@
         benchmarks to remediate high, and moderate risk flaws within a
         customer defined period after a flaws discovery.
         */'
+
+- control_key: SI-2(4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: complete
```

Comment:
```suggestion
  implementation_status: not applicable
```

Reply:
``complete`` generally means the control is satisfied through technical means or a process/procedure is documented.

Since this control was withdrawn, it should be considered ``not applicable`` to information systems moving forward. It's not a clean mapping -- ideally, we'd have some way to say "ignored" -- but for now, ``not applicable`` is what we got.

Reply:
@mammerman this should be swapped from ``complete`` to ``not applicable``